packages/teams-js/src/public/dialog.ts

```diff
@@ -18,6 +18,8 @@ import { runtime } from './runtime';
 export namespace dialog {
   /**
    * Data Structure to represent the SDK response when dialog closes
+   *
+   * @beta
```

Comment:
Since the branch was published, I thought we could not edit the file (or add beta tag) without bumping the version.  
Is this possible because @beta was added to the top of the file?

Reply:
Yes, I'm (very slightly) bending the rules here in the name of getting us into a consistent state. We already had the entire namespace marked with the beta tag, but that doesn't cause all of the functions within the namespace to get decorated correctly which could lead to some developer confusion. All of the contents of @dialog are in beta, so doing this will make it more likely for developers reading our documentation to get the right message